File: web/app/model/entity/Receipt.js
/*
 * Project: MajekWMS
 * User   : kornicameister
 * Date   : 04.11.12
 * Time   : 02:38
 */

Ext.define('WMS.model.entity.Receipt', {
    extend: 'WMS.model.entity.Invoice',
    proxy : {
        type       : 'wms',
        url        : 'wms/agent/invoice',
        extraParams: {
            type: 'receipt'
        }
    }
});

File: javaSrc/wms/model/Receipt.java
package wms.model;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;


@Entity
@DiscriminatorValue(value = "receipt")
public class Receipt extends Invoice {
    private static final long serialVersionUID = 5011897484071966063L;

    public Receipt() {
        super();
    }

}
